gocp/main.go

```diff
@@ -0,0 +1,48 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+gocp copies Go source code.
+The copied code is marked as generated.
+
+Usage of gocp:
+
+	-d, --dest string   The destination filepath.
+	-p, --pkg string    The destination package name (can contain comments).
+	-s, --src string    The source filepath.
+*/
+package main
+
+import (
+	"fmt"
+	"os"
+
+	flag "github.com/spf13/pflag"
```

Comment:
Why not just use [`flag`](https://pkg.go.dev/flag)?

Reply:
Only because this package is used in all other tools. Indirectly in most of them b/c of spf13/cobra and directly in `crosslilnk` and `semconvgen`. `flag` is not directly used in any of the tools.